core/module.h

```diff
@@ -12,7 +12,7 @@
 #include "task.h"
 
 static inline void set_cmd_response_error(pb_cmd_response_t *response,
-                                          const pb_error_t &error) {
+    const pb_error_t &error) {
```

Comment:
That doesn't seem like the clang-format output. Perhaps this file is missed? (just guessing)